test/modules/ModuleLib/ReporterClients/ModuleReporterClient.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.ModuleUtil.ReporterClients
+{
+    using System;
+    using System.Text;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil.TestResults;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    public sealed class ModuleReporterClient : ReporterClientBase
+    {
+        ILogger logger;
+        ModuleClient moduleClient;
+
+        private ModuleReporterClient(
+            ModuleClient moduleClient,
+            ILogger logger)
+            : base(logger)
+        {
+            this.logger = Preconditions.CheckNotNull(logger, nameof(logger));
+            this.moduleClient = Preconditions.CheckNotNull(moduleClient, nameof(moduleClient));
+        }
+
+        public static ModuleReporterClient Create(TransportType transportType, ILogger logger)
```

Comment:
Remove this; callers can do this by themselves

Reply:
No. I don't want the base class Create() to be async.